align/cell_fabric/gen_lef.py

```diff
@@ -89,6 +86,9 @@ def s(x):
     with open(input_json.parents[0] / macro_name, "wt") as fp:
 
         fp.write("MACRO %s\n" % out_lef)
+        fp.write("  UNITS \n")
+        fp.write("    DATABASE MICRONS UNITS %s ;\n" % (1000*Scale_factor))
+        fp.write("  END UNITS \n")
```

Comment:
@stevenmburns There might need some changes to the checker now because some changes are made in units and scale of input files.
LEF: values are now in the unit of nm (used to be in um).
pdk: actual value in nm is x/"ScaleFactor"
json: actual value in nm is x (used to be x/10, that means unit is 0.1nm).